### PR TITLE
#1088 included groovy scripts in jacoco coverage

### PR DESCRIPTION
**Pull request by krzyk**
#1088 
* removed exclusion for groovy scripts
* decreased coverages so the build passes (this will stop the coverage to drop)
* added todo to add more tests for stakeholders and increase the coverage
* added a missing test for changing of user rate

### CODE REVIEW

**Comment by 0crat** [comment]
Job [#1368](https://github.com/zerocracy/farm/issues/1368) is now in scope, role is `REV`

**Comment by 0crat** [comment]
This pull request [#1368](https://github.com/zerocracy/farm/issues/1368) is assigned to @amihaiemil[/z](https://www.0crat.com/u/amihaiemil), here is [why](https://www.0crat.com/footprint/C3NDPUA8L/1881201686); the budget is 15 minutes, see [§4](http://www.zerocracy.com/policy.html#4); please, read [§27](http://www.zerocracy.com/policy.html#27) and when you decide to accept the changes, inform @g4s8[/z](https://www.0crat.com/u/g4s8) (the architect) right in this ticket; if you decide that this PR should not be accepted ever, also inform the architect; this [blog post](http://www.yegor256.com/2015/02/09/serious-code-reviewer.html) will help you understand what is expected from a code reviewer; there will be a monetary reward for this job

File: src/test/resources/com/zerocracy/bundles/changes_user_rate/_after.groovy [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+import com.jcabi.xml.XML
+import com.zerocracy.Project
+import com.zerocracy.pm.cost.Rates
+import org.hamcrest.MatcherAssert
+import org.hamcrest.Matchers
+
+def exec(Project project, XML xml) {
+  MatcherAssert.assertThat(
+    new Rates(project).rate('user42').decimal().doubleValue(),
+    Matchers.equalTo(100.0D)
```
Comment (amihaiemil) [inline src/test/resources/com/zerocracy/bundles/changes_user_rate/_after.groovy]: @krzyk What is the initial rate of a user, can we make an assertion about that in the ``_before`` file?

File: src/test/resources/com/zerocracy/bundles/changes_user_rate/_before.groovy [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+import com.zerocracy.Farm
+import com.zerocracy.Project
+import com.zerocracy.pmo.People
+
+def exec(Project project, XML xml) {
+  Farm farm = binding.variables.farm
+  People people = new People(farm).bootstrap()
+  String user = 'user42'
+  people.wallet(user, 'paypal', 'user@example.com')
+  people.details(user, 'something')
```
Comment (amihaiemil) [inline src/test/resources/com/zerocracy/bundles/changes_user_rate/_before.groovy]: @krzyk Is it necessary to set the wallet and details for this test? If not, remove them, it's just noise I think.

File: src/test/resources/com/zerocracy/bundles/changes_user_rate/claims.xml [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+FITNESS FOR A PARTICULAR PURPOSE AND NON-INFRINGEMENT. IN NO EVENT SHALL THE
+AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
+SOFTWARE.
+-->
+<claims xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:noNamespaceSchemaLocation="http://datum.zerocracy.com/0.28/xsd/pm/claims.xsd" updated="2017-07-21T20:31:44.512Z" version="1">
+  <claim id="1">
+    <created>2017-07-21T20:32:19.716Z</created>
```
Comment (amihaiemil) [inline src/test/resources/com/zerocracy/bundles/changes_user_rate/claims.xml]: @krzyk Can you add a date closer to the present? I am thinking that, in the future, we may want to have 0crat ignore claims that are too old. This one is currently rather old :D

File: src/test/resources/com/zerocracy/bundles/changes_user_rate/roles.xml [hunk 4]
```diff
@@ -0,0 +1,25 @@
...
+OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
+SOFTWARE.
+-->
+<roles xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:noNamespaceSchemaLocation="http://datum.zerocracy.com/0.33/xsd/pm/staff/roles.xsd" version="0.1" updated="2017-08-07T10:37:00.000Z">
+  <person id="g4s8">
+    <role>ARC</role>
+  </person>
+  <person id="user42">
+    <role>DEV</role>
+  </person>
```
Comment (amihaiemil) [inline src/test/resources/com/zerocracy/bundles/changes_user_rate/roles.xml]: @krzyk Is it necessary to have a role for setting/changing the rate? I think that is the very first thing a user does when entering the system and by that time they have no roles.

File: src/test/resources/com/zerocracy/bundles/changes_user_rate/_before.groovy (hunk 2 above)
Comment (krzyk) [reply src/test/resources/com/zerocracy/bundles/changes_user_rate/_before.groovy]: @amihaiemil wallet is required, but details are not, so I'll remove those

**Comment by krzyk** [comment]
@amihaiemil updated, please take a look

**Comment by amihaiemil** [comment]
@krzyk looks good, thanks

**Comment by amihaiemil** [comment]
@rultor good to merge

**Comment by g4s8** [comment]
@rultor merge

**Comment by 0crat** [comment]
@ypshenychka[/z](https://www.0crat.com/u/ypshenychka) please review this job completed by @amihaiemil[/z](https://www.0crat.com/u/amihaiemil), as in [§30](http://www.zerocracy.com/policy.html#30); the job will be fully closed and all payments will be made when the quality review is completed

**Comment by 0crat** [comment]
The job [#1368](https://github.com/zerocracy/farm/issues/1368) is now out of scope

**Comment by 0crat** [comment]
Payment to `ARC` for a closed pull request, as in [§28](http://www.zerocracy.com/policy.html#28): +10 point(s) just awarded to @g4s8[/z](https://www.0crat.com/u/g4s8)

**Comment by ypshenychka** [comment]
@0crat quality good

**Comment by 0crat** [comment]
Order was finished, quality is "good": +38 point(s) just awarded to @amihaiemil[/z](https://www.0crat.com/u/amihaiemil)

**Comment by 0crat** [comment]
Quality review completed: +8 point(s) just awarded to @ypshenychka[/z](https://www.0crat.com/u/ypshenychka)

**Pull request merged**